DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/AddCentreCustomPromptsAccessibilityTests.cs

```diff
@@ -31,6 +31,10 @@ public void AddRegistrationPrompt_journey_has_no_accessibility_errors()
             ValidatePageHeading("Configure answers");
             var configureAnswerWithAnswersResult = new AxeBuilder(Driver).Analyze();
 
+            Driver.ClickButtonByText("Bulk add");
+            var bulkAdditionResult = new AxeBuilder(Driver).Analyze();

```

Comment:
Do you think it's worth checking page titles here too to ensure we're scanning the expected page? 🤔